spacepy/irbempy/NOTES.md

```diff
@@ -1,28 +1,43 @@
 # Notes on how to update the revision of IRBEMlib
+First you'll need to check out the IRBEM git repository from
+https://github.com/PRBEM/IRBEM
+Some required files get built by make, so in the IRBEM repo:
+```
+make OS=linux64 ENV=gnu64 all
```

Comment:
Ugh, do we pick up way too much junk in the repo this way? Any reason we shouldn't just update our build to make those? Can we do a make that just makes the necessary bits and not "all"? Or are we saved because we do a more careful git mv/replacing of files instead of just throwing the whole tree at it?

Reply:
Oh, if I'm reading this right, your compare script highlights the stuff that has to move, correct?

Reply:
We only track the source + a few add'l required files in the repo, so running make (when updating the IRBEM version) doesn't impact us from that perspective. What it does do is update the various `.inc` files and things that set version numbers, fill values, maxima for various array sizes, ... 

There is an argument to be made for cleaning up our build and including it there, but that's a pretty big scope creep for this PR. The reason this text is added to the notes here is because it wasn't documented previously, not because it's a new requirement.

Now, could we trim what we have in our repo? Yes. Do I think it needs to get done here? See my scope creep answer. This PR is just an update of the IRBEM source, primarily for bug fixes.